core/src/services/gcs/core.rs

```diff
@@ -236,13 +238,23 @@ impl GcsCore {
         if let Some(storage_class) = &self.default_storage_class {
             let mut multipart = Multipart::new();
 
+            let mut meta_data = HashMap::new();
+            meta_data.insert("storageClass".to_string(), storage_class.to_string());
+            meta_data.insert(
+                "cacheControl".to_string(),
+                cache_control
+                    .unwrap_or("public, max-age=3600")
+                    .parse()
+                    .unwrap(),
+            );
```

Comment:
I think it's better to keep it as optional here.

Reply:
ok